generators/app/index.js

```diff
@@ -346,7 +338,16 @@ module.exports = class extends BaseGenerator {
                 this.skipServer && (config.skipServer = true);
                 this.skipUserManagement && (config.skipUserManagement = true);
                 this.config.set(config);
+            },
+
+            insight() {
+                if (this.existingProject) return;
+
+                const yorc = Object.assign({}, this.configOptions);
+                yorc.applicationType = this.applicationType;
+                statistics.sendYoRc(yorc, this.jhipsterVersion);
```

Comment:
we shouldn't blindly send everything, we need to remove sensitive stuff like packageName, jwt token etc